### ISSUE TITLE
All day events are not displayed if there are a lot of them

### ISSUE THREAD

**Issue by tlozovyi** [thellmund/Android-Week-View#71]
6 all day events, 3 days shown in WeekView, all is ok: 
![device-2019-06-11-105344](https://user-images.githubusercontent.com/9742534/59253721-80974d00-8c37-11e9-9463-dd9deede4415.png)

6 all day events, 7 days shown in WeekView, no all day events:
![device-2019-06-11-105439](https://user-images.githubusercontent.com/9742534/59253750-8ee56900-8c37-11e9-8749-dfb22b90b934.png)

Tried setting 'columnGap' and 'overlappingEventGap' to 0 - no effect, there are still no all day events.

Here's my WeekView setup:
```xml    
<com.alamkanak.weekview.WeekView
        android:id="@+id/weekView"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_alignParentBottom="true"
        android:layout_below="@id/toolbar"
        app:columnGap="1dp"
        app:headerRowBottomLineColor="?settingsDividerColor"
        app:timeColumnSeparatorColor="?settingsDividerColor"
        app:daySeparatorColor="?settingsDividerColor"
        app:hourSeparatorColor="?settingsDividerColor"
        app:eventCornerRadius="4dp"
        app:eventTextColor="?textColorNormal"
        app:headerRowTextColor="?textColorNormal"
        app:todayHeaderTextColor="?colorAccent"
        app:timeColumnTextColor="?textColorNormal"
        app:eventTextSize="@dimen/little_text_size"
        app:hourHeight="60dp"
        app:overlappingEventGap="1dp"
        app:eventMarginVertical="1dp"
        app:singleDayHorizontalMargin="1dp"
        app:eventPadding="4dp"
        app:showCompleteDay="false"
        app:showNowLine="true"
        app:nowLineColor="?colorPrimary"
        app:dayBackgroundColor="@color/fui_transparent"
        app:todayBackgroundColor="@color/fui_transparent"
        app:headerRowBackgroundColor="@color/fui_transparent"
        app:timeColumnBackgroundColor="@color/fui_transparent"
        app:showTimeColumnSeparator="true"
        app:singleLineHeader="false"
        app:timeColumnPadding="8dp" />
```

**Comment by thellmund** [comment]
Are you using version 3.4.2 or a snapshot?

**Comment by tlozovyi** [comment]
3.4.2

**Comment by thellmund** [comment]
I’ll have a look in the next few days.

**Comment by tlozovyi** [comment]
Thanks! I'd really appreciate that!

**Comment by thellmund** [comment]
I’ve pushed a fix in d7417560a7f6ecbf662d8a4390f7c9de918b1fe6 that should address the issue. Feel free to check it out and let me know if anything works correctly. I intend to include it in the next release, version 4.
